dist/contract.js

```diff
@@ -122,39 +126,43 @@ var Contract = function () {
 
       var hex = void 0;
       _lodash2.default.each(methodObj.inputs, function (item, index) {
```

Comment:
do we hv `_lodash1` or `_encoder1`, if not, can we rename it?

Reply:
These are auto-generated by babel. As far as I see, it's always _lodash2.

Reply:
Yup auto-generated by babel when building.